src/main/java/com/alkemy/ong/web/document/EmailService.java

```diff
@@ -0,0 +1,48 @@
+package com.alkemy.ong.web.document;
```

Comment:
hay que seguir la estructura q teníamos. 
el controller en el web, con su dto. 
Dsp el servicio, que tenga el metodo sendMail(String to, String subject, String body).
Dsp definir la interfaz gateway (MailGateway) q va a tener un metodo sendMail(String to, String subject, String body).
Y el SendGridMailGateway va a tener esta lógica. Esta clase tiene q estar en un paquete mail a la misma altura de data y cloud. Asi podemos lograr el mismo efecto q con la base de datos. Si quisiéramos cambiar a otra q no sea send grid, podemos sin cambiar el domain